### PR TITLE
Changes/Fixes/Features to the Experimentor

### LINKED ISSUE #7224: EXPERIMENTOR creates a super capacitor, which gets instantly deleted
for posterity, it was a matter bin that turned into a super capacitor.

### LINKED ISSUE #7272: Experimenter Research Upgrade Malf Does Nothing Useful
Server: Basil

Revision: #7167

Problem Description: The Experimenter's chance to upgrade the item's research seems to not work at all.

What did you expect to happen: The thing gives higher research when scanned in the experimenter or Destructive Analyzer.

What happened instead: It gave no higher research, and didn't even show a number when put in the destructive analyzer.

Steps to reproduce the problem: Constantly test mess with something in the experimenter untill you get the Research Upgrade malfunction. Try to deconstruct it in the Destructive Analyzer.

Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?): If notable, I found this out with a Durand Weapons Control Board.

### PR DESCRIPTION
**Pull request by sawu-tg**
- Fixes #7224
- Re-fixes #7272
- Adds relics that spawn around the station, can be researched for randomised objects.
- Adds the meteor malfunction.

### CODE REVIEW

File: code/modules/research/experimentor.dm [hunk 1]
```diff
@@ -333,17 +344,20 @@
...
 			visible_message("<span class='notice'>[src]'s emergency coolant system gives off a small beep!</span>")
-			var/obj/item/weapon/reagent_containers/food/drinks/coffee/C = new /obj/item/weapon/reagent_containers/food/drinks/coffee(pick(oview(1,src)))
+			var/obj/item/weapon/reagent_containers/food/drinks/coffee/C = new /obj/item/weapon/reagent_containers/food/drinks/coffee(get_turf(pick(oview(1,src))))
 			var/list/chems = list("plasma","capsaicin","ethanol")
 			C.reagents.remove_any(25)
 			C.reagents.add_reagent(pick(chems),25)
 			C.name = "Cup of Suspicious Liquid"
 			C.desc = "It has a large hazard symbol printed on the side in fading ink."
 		if(prob(EFFECT_PROB_VERYLOW-badThingCoeff))
+			visible_message("<span class='notice'>[src] activates it's heat-seeking system!</span>")
```
Comment (GunHog) [inline code/modules/research/experimentor.dm]: This should REALLY be 'danger', not 'notice'.

**Comment by GunHog** [comment]
So, on top of EMP, fires, freezing, toxic gas, shooting your stuff back at you, it also causes *meteor storms*? I am loving this thing more and more.

**Comment by pudl** [comment]
haha, this is turning out to be really awesome, i'll give more ideas for relics on irc later

**Comment by Razharas** [comment]
Cmon sawu fix it

**Comment by Perakp** [comment]
Relics spawning on the station doesn't make sense, they should be on the asteroid/ other z levels.

**Comment by sawu-tg** [comment]
@Perakp Relic is the internal name, in game context, it's more "relic" as in old piece of junk found laying around, not sacred artifact. That's why they have names like "illegal technology" and "broken x". Stuff of shifts gone passed, left behind and forgotten in the tunnels.

**Comment by Razharas** [comment]
And i was just about to merge it, meh

**Pull request merged**